rq/dependency.py

```diff
@@ -0,0 +1,26 @@
+from .job import Job
+from redis.client import Pipeline
+from typing import List
+from redis.exceptions import WatchError
+
+
+class Dependency:
+    @classmethod
+    def get_jobs_with_dependencies_met(cls, jobs: List['Job'], pipeline: Pipeline):
```

Comment:
I think this should be `get_jobs_with_met_dependencies`?

Reply:
I renamed the function and relevant variables